packages/frontend/src/translations/en.global.json

```diff
@@ -975,6 +989,7 @@
         "switchAccounthNotAllowed": "This app doesn't allow account changes"
     },
     "sending": "Sending",
+    "importing": "Importing",
```

Comment:
There's some bug in `react-localize-redux` whenever the translation string contains `html` but there's no corresponding translations for other languages. It can cause error when switching to a language (not sure which browser) that doesn't have the specific string that contains `html` translated. The mitigation has been to simply copy-paste the string with `html` in the other `.json` files as a safety measure, and it shouldn't serve as a way to track which strings need translations. I grabbed the whole section instead of just the two strings just to avoid any confusion later :)

There's a possible [fix](https://github.com/ryandrewjohnson/react-localize-redux/issues/155) for this bug, but I'm not able to reproduce the bug anymore and hence not able to verify the fix works :/

Reply:
FYI, I implemented the fix from the thread you linked, which might be why you can't reproduce it anymore :D

Reply:
@MaximusHaximus oh well 😆 ❤️